code/game/objects/items/devices/modificationkit.dm

```diff
@@ -0,0 +1,23 @@
+/obj/item/modkit
+	name = "modification kit"
+	desc = "A one-use kit, which enables kinetic accelerators to be fired with only one hand, as well as allowing less dextrous races to use the tool."
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "modkit"
+	origin_tech = "programming=2;materials=2;magnets=4"
+	var/uses = 1
+
+/obj/item/modkit/afterattack(var/obj/item/weapon/gun/energy/kinetic_accelerator/C, mob/user)
```

Comment:
no var/ in proc arguments

just

/obj/item/weapon/gun/energy/kinetic_accelerator/C

I wish I could just make these changes and merge it myself I think there is a way to do it